plugin.video.massengeschmack/.gitignore

```diff
@@ -0,0 +1,27 @@
+# OS-specific files
```

Comment:
I just feel like this whole process produces more work than necessary. I have to copy over all the stuff from the main repository and create a new commit. Having at least a .gitignore would make sure that no IDE files and other unwanted stuff ends up in that commit by accident.